tools/styles/package.json

```diff
@@ -110,7 +110,7 @@
     "devDependencies": {
         "@spectrum-css/commons": "^4.0.2",
         "@spectrum-css/expressvars": "^2.0.3",
-        "@spectrum-css/tokens": "^7.0.0",
+        "@spectrum-css/tokens": "^8.0.2",
```

Comment:
Looks like this was handled in https://github.com/adobe/spectrum-web-components/pull/3102/files#diff-fc30c96378462dd0d14f5f111c422e0f1d23749286f32f109173accff3eb5bb6R113, are there other changes you'd like to address with this PR, ro can we close?